Transformer: labels to fields should not also merge frames (#38671)
grafana/grafana: CHANGELOG.md

8.2.0...

> ### Breaking changes

Author: is this where we define breaking change text?

Reviewer: I'm not sure 🙈 @torkelo do you know?

Author: @achatterjee-grafana any thoughts on how to best do this?

I also added links, but not sure that is the right syntax :)

Reviewer 2: I think this is the right place to announce breaking changes. See https://github.com/grafana/grafana/blob/main/CHANGELOG.md#breaking-changes-3.

Tagging @pkolyvas.

Author: Looks like the script will read from text in the issue summary -- I moved this text to that section.  Thanks!

> The "labels to fields" transformer was changed so that label values are ordered after the value and multiple frames are not

Reviewer: Should we mention that if anyone still want to keep the auto-merge they can just add a merge transformation after this transformation.

8.2.0...

### Breaking changes

> The [labels to fields](https://grafana.com/docs/grafana/latest/panels/transformations/types-options/#labels-to-fields) transformer was changed so that frames are not automatically merged into a single frame. To reproduce
> this behavior, simply add an additional [merge](https://grafana.com/docs/grafana/latest/panels/transformations/types-options/#merge) transformation after the [labels to fields](https://grafana.com/docs/grafana/latest/panels/transformations/types-options/#labels-to-fields) step. Any dashboards saved
> before 8.2 will have an additional [merge](https://grafana.com/docs/grafana/latest/panels/transformations/types-options/#merge) transformation applied automatically.

Reviewer 2: ```suggestion
The [labels to fields](https://grafana.com/docs/grafana/latest/panels/transformations/types-options/#labels-to-fields) transformer was changed so that frames are no longer automatically merged into a single frame. To reproduce this behavior, add a [merge](https://grafana.com/docs/grafana/latest/panels/transformations/types-options/#merge) transformation after the [labels to fields](https://grafana.com/docs/grafana/latest/panels/transformations/types-options/#labels-to-fields) step. 

Dashboards saved before the Grafana 8.2  release will have an additional [merge](https://grafana.com/docs/grafana/latest/panels/transformations/types-options/#merge) transformation applied automatically.
```
